### PR TITLE
update to load smd2023 report

### PR DESCRIPTION
**Pull request by matthewdylan**
updating a few fields so we can load the smd2023 report

* path var updated to point to Salesforce ID: 00O6f000007v2YZEAY
* array instance updated to 3 (in line with where we usually find the info we need for SMD/FMD)
* push_to_s3's filename var updated to smd2023.json

### CODE REVIEW

File: get-report.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 sf = SalesforceConnection()
 
-path = "/services/data/v46.0/analytics/reports/00O6f000007udDgEAI"
+path = "/services/data/v46.0/analytics/reports/00O6f000007v2YZEAY"
 url = "{}{}".format(sf.instance_url, path)
 resp = requests.get(url, headers=sf.headers)
 content = json.loads(resp.text)
 print(content)
-the_good_stuff = content["factMap"]["T!T"]["aggregates"][0] # change this to 3 for SMD/FMD
+the_good_stuff = content["factMap"]["T!T"]["aggregates"][3] # change this to 0 for Giving Tuesday
```
Comment (ashley-hebler) [inline get-report.py:14]: **suggestion: (non-blocking)** This comment is a funny system of reminding ourselves of how this works 😄. I know it's a pattern you're just building on so nothing wrong with this, but maybe we just keep both comments?

```
# use 0 for Giving Tuesday
# use 3 for SMD/FMD
```

It's a little messy but makes it so we don't have to keep changing it back and forth. 😵

**Review by ashley-hebler — APPROVED** [review]
Left a small note, but overall LGTM!

File: get-report.py (hunk 1 above)
Comment (matthewdylan) [reply get-report.py]: Good call... just updated and moved some things around/removed some magic numbers... think it's easier to see what pieces can be updated now

Comment (ashley-hebler) [reply get-report.py]: Ah yes, much easier to follow! Thank you @matthewdylan

**Pull request merged**